src/Themes/NexusMods.Themes.NexusFluentDark/Controls/Button.axaml

```diff
@@ -0,0 +1,96 @@
+      <Setter Property="BorderBrush" Value="{StaticResource ButtonBorderBrushPointerOver}" />
+      <Setter Property="Foreground" Value="{StaticResource ButtonForegroundPointerOver}" />
+    </Style>
+
+    <Style Selector="^:pressed">
+        <!-- unused -->
+        <!-- <Setter Property="RenderTransform" Value="scale(0.98)" /> -->
+    </Style>
+
+    <Style Selector="^:pressed  /template/ ContentPresenter#PART_ContentPresenter">
+      <Setter Property="Background" Value="{StaticResource ButtonBackgroundPressed}" />
+      <Setter Property="BorderBrush" Value="{StaticResource ButtonBorderBrushPressed}" />
+      <Setter Property="Foreground" Value="{StaticResource ButtonForegroundPressed}" />
+    </Style>
+
+    <Style Selector="^:disabled /template/ ContentPresenter#PART_ContentPresenter">
+      <Setter Property="Background" Value="{StaticResource ButtonBackgroundDisabled}" />
+      <Setter Property="BorderBrush" Value="{StaticResource ButtonBorderBrushDisabled}" />
+      <Setter Property="Foreground" Value="{StaticResource ButtonForegroundDisabled}" />
+    </Style>
+
+    <!-- unused -->
+    <!-- <Style Selector="^.accent"> -->
+    <!--   <Style Selector="^ /template/ ContentPresenter#PART_ContentPresenter"> -->
+    <!--     <Setter Property="Background" Value="{StaticResource AccentButtonBackground}" /> -->
+    <!--     <Setter Property="BorderBrush" Value="{StaticResource AccentButtonBorderBrush}" /> -->
+    <!--     <Setter Property="Foreground" Value="{StaticResource AccentButtonForeground}" /> -->
+    <!--   </Style> -->
+    <!-- -->
+    <!--   <Style Selector="^:pointerover /template/ ContentPresenter#PART_ContentPresenter"> -->
+    <!--     <Setter Property="Background" Value="{StaticResource AccentButtonBackgroundPointerOver}" /> -->
+    <!--     <Setter Property="BorderBrush" Value="{StaticResource AccentButtonBorderBrushPointerOver}" /> -->
+    <!--     <Setter Property="Foreground" Value="{StaticResource AccentButtonForegroundPointerOver}" /> -->
+    <!--   </Style> -->
+    <!-- -->
+    <!--   <Style Selector="^:pressed  /template/ ContentPresenter#PART_ContentPresenter"> -->
+    <!--     <Setter Property="Background" Value="{StaticResource AccentButtonBackgroundPressed}" /> -->
+    <!--     <Setter Property="BorderBrush" Value="{StaticResource AccentButtonBorderBrushPressed}" /> -->
+    <!--     <Setter Property="Foreground" Value="{StaticResource AccentButtonForegroundPressed}" /> -->
+    <!--   </Style> -->
+    <!-- -->
+    <!--   <Style Selector="^:disabled /template/ ContentPresenter#PART_ContentPresenter"> -->
+    <!--     <Setter Property="Background" Value="{StaticResource AccentButtonBackgroundDisabled}" /> -->
+    <!--     <Setter Property="BorderBrush" Value="{StaticResource AccentButtonBorderBrushDisabled}" /> -->
+    <!--     <Setter Property="Foreground" Value="{StaticResource AccentButtonForegroundDisabled}" /> -->
+    <!--   </Style> -->
+    <!-- </Style> -->
```

Comment:
I'm guessing these are from Avalonia styles?
We would normally want to clear out dead code, but if this makes it easier to work with styles I'm fine with keeping it.